DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py

```diff
@@ -4,7 +4,7 @@
 
 process = cms.Process("PIXELDQMLIVE", eras.Run2_2017)
 
-live=True  #set to false for lxplus offline testing
```

Comment:
@akhatiwa live needs to be set at true, please fix it and push again (also in the 10_1_X PR)